web/pageComponents/shared/textTeaser/TextTeaser.tsx

```diff
@@ -0,0 +1,123 @@
+import { Teaser as EnvisTeaser, Link, BackgroundContainer } from '@components'
+import styled from 'styled-components'
+import IngressText from '../portableText/IngressText'
+import TitleText from '../portableText/TitleText'
+import { getUrlFromAction } from '../../../common/helpers/getUrlFromAction'
+
+import type { TextTeaserData, LinkData } from '../../../types/types'
+import { getLocaleFromName } from '../../../lib/localization'
+import { getColorForTheme } from './theme'
+import { CSSProperties } from 'react'
+
+const { Content } = EnvisTeaser
+
+type TitlePostion = 'left' | 'right' | undefined
+
+type TextTeaserProps = {
+  data: TextTeaserData
+  anchor?: string
+}
+
```

Comment:
somewhere in your styles an extra padding is being added to the last component:

<img width="621" alt="image" src="https://github.com/equinor/energyvision/assets/23337266/07492d83-518c-47d4-ad85-81d979ce0514">